BasePolicies/CODE_OF_CONDUCT.md

```diff
@@ -1,27 +0,0 @@
-## Working Group Code of Conduct Template
```

Comment:
Are these "BasePolicy" files in the nodejs/admin or somewhere else now?

Reply:
Bootstrap governance docs are at https://github.com/nodejs/TSC/blob/main/WORKING_GROUPS.md#bootstrap-governance

Reply:
Fair enough. I guess the `Application/*` files are replaced by https://github.com/nodejs/TSC/blob/main/WORKING_GROUPS.md#current-working-groups sections as well